_data/accessibility-tests/identifier.yml

```
@@ -0,0 +1,73 @@
+title: Identifier
+component_status: pass
+test_items:
+# General tests
+  - summary: Identifier appears consistently. 
+    summary_additional: The identifier appears in the same location across all pages of the website.
+    test_status: conditional
+    test_type: general
+    version_tested: 3.8.1
+    wcag_criterion: 3.2.3
+# Zoom/screen magnification tests
+  - summary: Identifier links are visible and functional when screen is magnified.
+    summary_additional: When you zoom in to 400%, identifier links are stacked and visible. No left-to-right scrolling is required.
+    test_status: pass
+    test_type: zoom
+    version_tested: 3.8.1
+    wcag_criterion: 1.4.10
+  - summary: Identifier content is functional when screen is resized. 
+    summary_additional: When you zoom in to 200%, other content does not block or overlap the identifier content.
+    test_status: pass
+    test_type: zoom
+    version_tested: 3.8.1
+    wcag_criterion: 1.4.4
+# Keyboard navigation tests
+  - summary: Keyboard navigates through links in logical order.
+    summary_additional: When tabbing through the identifier, you can access the links in a logical reading order.
+    test_status: pass
+    test_type: keyboard
+    version_tested: 3.8.1
+    wcag_criterion: 1.3.2
+  - summary: Keyboard navigates through all elements in identifier.
+    summary_additional: When you navigate through the identifer with a keyboard, you can interact with every element inside the identifier.
+    test_status: pass
+    test_type: keyboard
+    version_tested: 3.8.1
+    wcag_criterion: 2.1.1
+  - summary: Focus indicator is clearly visible. 
+    summary_additional: When you use a keyboard to focus on identifier elements, will have a visible outline or other clear indication of focus. 
```

Gherkin language was incomplete. I would correct to say "When you use a keyboard to focus on identifier elements, you will see a visible outline or other clear indication of focus." @alex-hull  or @amycole501 can you confirm this is correct? @amyleadem maybe confirm too?

---

Yes that's correct

---

Looks like it's just missing the `you`

```suggestion
    summary_additional: When you use a keyboard to focus on identifier elements, you will have a visible outline or other clear indication of focus. 
```

---

Looks good to me. Going to resolve the comment.